rust/src/web/graphql/transactions/mod.rs

```diff
@@ -31,14 +30,25 @@ impl TransactionsQueryRoot {
         ctx: &Context<'_>,
         query: TransactionQueryInput,
     ) -> Result<Option<Transaction>> {
+        let db = db(ctx);
         if let Some(hash) = query.hash {
             if signed::is_valid_tx_hash(&hash) {
-                let db = db(ctx);
                 return Ok(db
                     .get_command_by_hash(&hash)?
                     .map(|cmd| txn_from_hash(cmd, db)));
             }
+        } else {
+            // no query filter => return the most recent transaction
```

Comment:
This seems like functionality that should be documented (more so than in just a code comment).

Reply:
Agreed. Is this the expected behavior for query `transactions {...}`?

Reply:
Or should we return the default limit number of transactions?

Reply:
@Isaac-DeFrain, I would think return the default limit number of transactions. I made an issue: https://github.com/Granola-Team/mina-indexer/issues/805